.husky/pre-commit

```diff
@@ -1,9 +1,11 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
+project_root=$(pwd)
 
 npx is-ci && exit 0
+cd ${project_root}/packages/askui-nodejs
 npm run lint:staged
 npm test
 
-cd examples/askui-nodejs-base-example || exit
+cd ${project_root}/examples/askui-nodejs-base-example || exit
 npm run lint:staged
```

Comment:
If you remove `examples`, don't forget to remove this, too, and other places where we reference it.